contrib/buildrpm/README

```diff
@@ -0,0 +1,89 @@
+
+# Copyright (c) 2016      Cisco Systems, Inc.  All rights reserved.
+#
+# Written by: Landon Curt Noll, Martin Kontsek
```

Comment:
I am sorry, this is cultural thing for me, as I was taught to give the credit to other people first.